reconcile/gitlab_housekeeping.py

```
@@ -1085,6 +1086,22 @@ def _process_omm_group(
     any_active = False
 
     for mr in pending:
+        if any(label in ERROR_LABELS for label in mr.labels):
+            logging.info([
+                "omm-group",
+                "eject-error-label",
+                gl.project.name,
+                mr.iid,
+                [l for l in mr.labels if l in ERROR_LABELS],
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)